NLog.Etw/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.Diagnostics.Tracing.EventSource.Redist" version="1.1.28" targetFramework="net45" />
```

Comment:
NuGet pack was failing due to the content files reference. The package dependencies (NLog, Microsoft.Diagnostics.Tracing.EventSource.Redist) are added automagically.